terraform/environment/locals.tf

```diff
@@ -31,7 +31,7 @@ locals {
   account_name      = lookup(var.account_mapping, terraform.workspace, "development")
   account           = var.accounts[local.account_name]
   environment       = lower(terraform.workspace)
-  dns_namespace_env = local.account_name != "development" ? "" : "${local.environment}."
+  dns_namespace_env = local.account_name == "production" ? "" : "${local.environment}."
```

Comment:
😢

Reply:
This conditional will disappear after ecs migration